ansible/roles/ansible-horizon-plugin/tasks/uninstall_rhel.yml

```diff
@@ -4,19 +4,30 @@
      update_cache: yes
      name:
         - tvault-horizon-plugin
-        - python-workloadmgrclient
+        - workloadmgrclient
```

Comment:
mentioning wlmclient should not be required. uninstalling horizon-plugin should uninstall all the dependent pkgs.

Reply:
workloadmgrclient package removed from  code